rosdep/python.yaml

```diff
@@ -447,6 +447,26 @@ pyros-setup-pip:
   ubuntu:
     pip:
       packages: [pyros-setup]
+pysdl2:
+  alpine:
+    pip:
+      packages: [pysdl2]
+  debian:
+    pip:
+      packages: [pysdl2]
+  ubuntu:
+    pip:
+      packages: [pysdl2]
+pysdl2-dll:
+  alpine:
+    pip:
+      packages: [pysdl2-dll]
+  debian:
+    pip:
+      packages: [pysdl2-dll]
+  ubuntu:
+    pip:
+      packages: [pysdl2-dll]
```

Comment:
Please add Alpine package to aports-ros-experimental if possible. (Version of the package won't be controlled via pip)

Reply:
I add the backports for `pysdl2-dll` and `pysdl2` in https://github.com/seqsense/aports-ros-experimental/pull/713.

Reply:
Updated in bc1d41f54400fd2fde26ed6da3b9ca333c135a9f